arrow-fx-coroutines-stream/src/test/kotlin/arrow/fx/coroutines/stream/CallbackTest.kt

```diff
@@ -206,17 +203,17 @@ class CallbackTest : StreamSpec(iterations = 250, spec = {
   }
 })
 
-private fun <A> countToCallback(
+private suspend fun <A> countToCallback(
   iterations: Int,
   map: (Int) -> A,
   cb: suspend (A) -> Unit,
   onEnd: suspend () -> Unit = { }
-): Unit = suspend {
+): Unit {
   var i = 0
   arrow.fx.coroutines.repeat(Schedule.recurs(iterations)) {
```

Comment:
Yes, it could be a method on `Schedule` itself.

```
Schedule.recurs(iterations).repeat {
  ...
}
```

Only annoying is that in compositions you'd need to add parenthesis around it.

`(Schedule.recurs(iterations) zipRight Schedule.identity()).repeat { ... }` it might also worsen type inference since now `repeat` can infer which type `Schedule` has  to be based on the return type of lambda. Which is not possible in the  latter case, right? 🤔